projects/gloo/cli/pkg/cmd/install/installer.go

```
@@ -101,11 +112,14 @@ func (i *installer) Install(installerConfig *InstallerConfig) error {
 	}
 
 	// determine if it's an enterprise chart by checking if has gloo as a dependency
-	installerConfig.Enterprise = false
-	for _, dependency := range chartObj.Dependencies() {
-		if dependency.Metadata.Name == constants.GlooReleaseName {
-			installerConfig.Enterprise = true
-			break
+	// if so, overwrite the installation mode to Enterprise
+	if installerConfig.Mode != Federation {
```

we don't want this check. this check allows users to provide a helm chart install via -f flag and automatically determine if we have an enterprise chart even if we're installing from a different sub command.

perhaps better UX here would be to loudly fail rather than silently fix the helm values we pass through

this change allows users to `glooctl install gateway -f <helm glooe chart` and have this succeed. with the check above we could not do the same with federation